### PR TITLE
Feat: Cart jsx

### PR DESCRIPTION
**Pull request by sri-rishi**
### files to check

- Cart.jsx
- HorizontalCard.jsx
- PriceSummary.jsx

Cart page

image

### CODE REVIEW

**Review by Naman-Saxena1 — APPROVED** [review]
Code looks good to me, nice work 👍

File: src/pages/Cart/components/HorizontalCard/HorizontalCard.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { AiOutlineMinus, AiOutlinePlus} from "../../../../assets/icons"
+import { Button } from "../../../../Components/index"
```
Comment (nikhiljugale007) [inline src/pages/Cart/components/HorizontalCard/HorizontalCard.jsx]: `import { Button } from "../../../../Components"
`

This will also work

**Review by nikhiljugale007 — COMMENTED** [review]
Added one comment in code.
Ui is clean and code. Components are also small.

File: src/pages/Cart/components/HorizontalCard/HorizontalCard.jsx (hunk 1 above)
Comment (sri-rishi) [reply src/pages/Cart/components/HorizontalCard/HorizontalCard.jsx]: It's auto imported

**Pull request merged**